docs/advanced/full-node.md

```diff
@@ -0,0 +1,63 @@
+# <img class="dcr-icon" src="/img/dcr-icons/AtoB.svg" /> Running a Full Node
+
+---
+
+This guide explains everything you need to know about running a full node on Decred.
+
+## What is a full node?
+
+A full node is a program that fully validates transactions and blocks, without having to rely on a third party. For this guide, we'll be using [dcrd](https://github.com/decred/dcrd), the most popular full node implementation for Decred. 
+
+## Why run a full node?
+
+There are a number of reasons to run a full node:
+
+- **Help the network:** Full nodes are the backbone of Decred. By validating and broadcasting transactions without relying on third parties&mdash; as lightweight nodes do&mdash;they help keep the network decentralized. Each full node added to the network adds to its security and robustness. 
+- **Safety & Privacy:** Because a lightweight node cannot fully validate transactions and blocks, it must trust third parties to do so. This means that the third party will have access to information about your transactions, as well as any personal data you give to that third party. For instance, many lightweight wallets rely on a centralized server to process transactions. Even Simplified Payment Verification (SPV) wallets, which typically connect directly to full nodes in the network, still rely on those other full nodes to process and broadcast their transactions. 
+
+## Requirements:
+
+Below are the basic requirements for running a full node:
+
+- \>15 GB disk space (as of September 2019, increases over time)
```

Comment:
Linking to a blockchain size chart might be a reasonable idea, but it would be far more misleading without additional information than this value (which is intentionally high as discussed in a moment).  The problem is that the actual space is quite a bit more than the raw sum of the blocks.  There is a lot of metadata in addition to the raw blocks such as the spend journal, the utxo set, the upcoming required version 2 block filters, and optionally more depending on what indexes the user has chosen to enable.

Also, the value provided needs to be at least 2x whatever the current usage is in general to better support migrations and future growth, because remember that it gets larger every day, so if you actually recommend the current space, they're already going to be about of disk space in short order.  That is why it currently recommends around 3x the current raw blockchain size.

Given all of that, if we want a more dynamic approach it will need to say something like "3 times the current raw size reported at [blockchain size chart](https://dcrdata.decred.org/charts?chart=blockchain-size)".  That would obviously need to be worded better, but you get the idea.

Reply:
Updated to this:

>- \>15 GB disk space (as of September 2019). 

>!!! note "Note"
    The size of the blockchain is continually increasing as new blocks are added to the chain. To support migration to new versions and future growth, a good rule of thumb is to have ~3X the [current raw blockchain size](https://dcrdata.decred.org/charts?chart=blockchain-size&zoom=ikd7pc00-k0vn9c00&bin=day&axis=time) in hard disk space.

Reply:
Looks good to me. We should remove the extra params (`&zoom=ikd7pc00-k0vn9c00&bin=day&axis=time`) from the url so the chart loads with the default zoom etc.

Reply:
Done